Can't install realm-gradle-plugin:2.3.0-SNAPSHOT from OJO

When trying to use the latest snapshot version (2.3.0-SNAPSHOT), the assembly fails with:

```
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-gradle-plugin/2.3.0-SNAPSHOT/realm-gradle-plugin-2.3.0-20161219.025837-22.pom                                                                                 
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-transformer/2.3.0-SNAPSHOT/realm-transformer-2.3.0-20161219.025903-20.pom                                                                                     
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-annotations/2.3.0-SNAPSHOT/realm-annotations-2.3.0-20161219.025835-22.pom                                                                                     
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-gradle-plugin/2.3.0-SNAPSHOT/realm-gradle-plugin-2.3.0-20161219.025837-22.jar                                                                                 
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-transformer/2.3.0-SNAPSHOT/realm-transformer-2.3.0-20161219.025903-20.jar                                                                                     
Download http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-annotations/2.3.0-SNAPSHOT/realm-annotations-2.3.0-20161219.025835-22.jar                                                                                     

FAILURE: Build failed with an exception.                                                                             

* What went wrong:                                                                                                   
A problem occurred configuring project ':app'.                                                                       
> Could not resolve all dependencies for configuration ':app:_PRODDebugApkCopy'.                                     
   > Could not find io.realm:realm-android-library:2.3.0-SNAPSHOT.                                                   
     Required by:                                                                                                   
         miki-android:app:unspecified                                                                               
   > Could not find io.realm:realm-annotations:2.3.0-SNAPSHOT.                                                       
     Required by:                                                                                                   
         miki-android:app:unspecified   
```

I do point to `maven { url 'http://oss.jfrog.org/artifactory/oss-snapshot-local' }` in my Gradle configuration.

I did try with older snapshots version as well (2.2.0 and 2.0.0) and got exactly the same result...

I am very interested in getting Realm 2.2.2 and above because of the removal of final modifier on most classes which would allow using Mockito as I've tried writing UTs with Robolectric and PowerMock and it was a horrible and fragile thing.

--- a/README.md
+++ b/README.md
@@ -42,9 +42,11 @@ buildscript {
     }
 }
 
-repositories {
-    maven {
-        url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
+allprojects {
+    repositories {
+        maven {
+            url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
+        }
     }
 }
 ```

